### ISSUE TITLE
how to customize UI ?

### ISSUE THREAD

**Issue by echolove38** [SVG-Edit/svgedit#698]
the editor's html and js is too coupling, it's hard for users to customize UI.
Do u have the plan to refactor the editor.

**Comment by jfhenon** [comment]
We did a lot of refactoring already but I agree with your statement. Customizing the UI requires a good level of understanding.
What type of refactoring would you suggest?

**Comment by echolove38** [comment]
> 

I think the html dom should't mixed with js code, then I will customize the UI easily

**Comment by jfhenon** [comment]
Obviously, we will keep improving SvgEdit and the way to customize the UI. 
On your side, don't hesitate to submit PRs.
